packaging/calculix-precice_2.17-1_amd64/usr/share/doc/calculix-precice/copyright

```diff
@@ -0,0 +1,9 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: preCICE
+Upstream-Contact: preCICE mailing-list <info@precice.org>
+Copyright: (c) 2019 the preCICE developers <info@precice.org>
+Source: https://github.com/precice/precice
+
+Files: *
+Copyright: (c) 2019 the preCICE developers <info@precice.org>
```

Comment:
Now I am wondering if we should update the date every now and then.
We should also mention somewhere the license of CalculiX.